### PR TITLE
feat(livestock-animal-set-leave): add animal set leave documentation

### PR DESCRIPTION
**Pull request by ryanjosephtorrado**
Ticket: [AB#19805](https://dev.azure.com/rezaresystems/15020b92-fee6-4202-956f-9036d1e1da84/_workitems/edit/19805)

----

Notes
- add the `livestock-animal-set-leave` documentation

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [ ] Images stored in `_images`
- [x] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/index.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ Currently supported Resource Types are below:
     Animal provides information about individual animals.
   - [Animal Sets](/resource-types/livestock/animal-set.md)
 	Defines a set of identified animals which may be created for a particular purpose.
+  - [Animal Set Joins](/resource-types/livestock/animal-set-join.md)
```
Comment (danielR9) [inline docs/resource-types/index.md:61]:
```suggestion
  - [Animal Set Join Events](/resource-types/livestock/animal-set-join.md)
```
Not sure about the wording of `Animal Set Joins` vs `Join Events`
@chriiiish what do you think on the wording?
May want to change this in all the titles

File: docs/resource-types/livestock/animal-set-join.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+# Resource Type: Livestock - Animal Set Joins
+
+- [URLs](#urls)
+- [Response Structure](#response-structure)
+
+---
+
+## URLs
+
+Get all animal set joins that you have access to
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-set-join.md:23]: As above - maybe "get all animal set join events"

File: docs/resource-types/livestock/animal-set-leave.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+| **Location**           | An identifier for the location of the animal set leave                                                 | [Identifier](/resource-types/common.md#identifier) |
+| **Meta**               | Meta data for the resource                                                                             | [Metadata](/resource-types/common.md#metadata)     |
+| **Resource Type**      | The fixed discriminator for the animal set leave resource type.<br/>Value: /livestock/animal-set-leave | String                                             |
+| **Contemporary Group** | A contemporary group code that would affect statistical analysis                                       | String                                             |
+| **Event Date Time**    | The time the event occurred                                                                            | Datetime                                           |
+| **Id**                 | The Pure Farming Id of this animal set join                                                            | UUID                                               |
```
Comment (danielR9) [inline docs/resource-types/livestock/animal-set-leave.md:78]:
```suggestion
| **Id**                 | The Pure Farming Id of this animal set leave event | UUID                                               |
```

**Review by danielR9 — COMMENTED** [review]
Generally not sure about "joins" vs "join events" (and also singular "join event" when in sentences)

**Comment by ryanjosephtorrado** [comment]
@danielR9 I've updated the name and titles. Please check and review. Thank you

**Review by danielR9 — APPROVED** [review]

**Pull request merged**